test/test-cases.setup.js

```diff
@@ -0,0 +1,40 @@
+import { basename, dirname, join } from 'path';
+import webpack from 'webpack';
+import I18nPlugin from '../src';
+
+export default function processFile(entry, ...pluginOpts) {
+  const resolvedEntry = join(__dirname, 'test-cases', entry);
+  const resolvedOutput = join(dirname(resolvedEntry), `${basename(resolvedEntry, '.code.js')}.tmp.js`);
```

Comment:
I'm also struggling with how to setup tests _at best_ in a generic way for 'architectural' consistency (where possibel) across the org, don't mind me too much in the scope of this PR, the current is totally 👍